orc8r/gateway/python/magma/configuration/mconfig_managers.py

```diff
@@ -172,6 +192,21 @@ def load_service_mconfig(
         service_mconfig = mconfig.configs_by_key[service_name]
         return unpack_mconfig_any(service_mconfig, mconfig_struct)
 
+    def load_shared_mconfig(
```

Comment:
Can we suppress this pep8 warning and say the docs are in the abstract method

Reply:
Disabled the missing-docstrings warning for the entire class since all methods are already documented in base class. Would that be a reasonable alternative?